tests/baselines/reference/variadicTuples2.errors.txt

```diff
@@ -204,9 +200,19 @@ tests/cases/conformance/types/tuple/variadicTuples2.ts(130,25): error TS2345: Ar
         ~
 !!! error TS2322: Type '[number, ...T]' is not assignable to type '[number, ...number[]]'.
 !!! error TS2322:   Type at position 1 in source is not compatible with type at position 1 in target.
-!!! error TS2322:     Type 'T[number]' is not assignable to type 'number'.
+!!! error TS2322:     Type 'T' is not assignable to type 'number[]'.
+!!! error TS2322:       Type 'unknown[]' is not assignable to type 'number[]'.
+!!! error TS2322:         Type 'unknown' is not assignable to type 'number'.

```

Comment:
As far as I can tell this is still correct, the reported error is now slightly different cause of the change in the logic (position by position comparison instead of comparing an indexed slice of the source to the target type)